File: LinearSearch/linearSearch.js

    //O(n)
    
   


const linersearchalgo = (arr,target) => {
    for ( let i=0; i < arr.length; i++ ){
        if (arr[i] === target){
            return i
        }
    }

    return null
}

let results = linersearchalgo([ 3,4,5,6], 4)
console.log(results);
 
const verifieed = (results) => {
    ( results != null) ? console.log("verified") : console.log("Not verified");
}
verifieed( results )

File: LinearSearch/linearSearch.py
##linear search
def linear_search(arr,target):
    
    for i in range(0,len(arr)):
        if arr[i] == target:
            return i
    return None

def verify(index):
    if index != None:
        print(f'the index is {index} and is verified')
    else:
        print("Target is not found in list")


list=[1,3,4,8,9,34]
index=linear_search(list,4)
verify(  index  )
       